project/plugins.sbt

```diff
@@ -26,3 +26,5 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.4")
 
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
+
+addDependencyTreePlugin
```

Comment:
Seemed to need this in order to successfully use `dependencyBrowseTree`

Reply:
We removed `addDependencyTreePlugin` in this PR #600 because snyk does not support it.